app/usecases/localisation/localisation.go

```diff
@@ -12,12 +16,43 @@ func T(c *gin.Context, s string, args ...interface{}) string {
 	return localeEngine.Get(GetLang(c), s, args...)
 }
 
+func getQuality(s string) float32 {
```

Comment:
Moved this stuff in from /internal/locale/parse_header.go, and removed it; seemed like superfluous abstraction